src/styles/card.scss

```diff
@@ -0,0 +1,102 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 18px;
+  color: $main-text-color;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  padding: 16px;
```

Comment:
it is better to use `padding: 32px 16px 16px` and not to set margin top for image